tests/unicode.php

```diff
@@ -0,0 +1,39 @@
+--TEST
+Test V8::executeString() : Check if imported code works with umlauts
+--SKIPIF--
+<?php require_once(dirname(__FILE__) . '/skipif.inc'); ?>
+--FILE--
+<?php
+
+# maybe more characters (e.g. from http://www.ltg.ed.ac.uk/~richard/unicode-sample.html?)
+$unicode = 'äöüßÜÄÖÜß€áàâÁÀÂµ²³▁▂▃▄▅▆▇█    ㌀ ㌁ ㌂ ㌃';
+
+# insert unicode via snapshot
+$snapshot = V8Js::createSnapshot("var snapshot = {unicode: '" . $unicode . "'}");
+
+# start V8Js
+$jscript = new V8Js('php', array(), array(), true, $snapshot);
+
+# insert unicode via php var
+$jscript->unicode = $unicode;
+
+# insert unicode via executeString
+$jscript->executeString("var execStr = {unicode: '" . $unicode . "'}");
+
+# return  to php
+$jscript->executeString("values = {}");
+$jscript->executeString("values['snapshot'] = snapshot.unicode");
+$jscript->executeString("values['php'] = php.unicode");
+$jscript->executeString("values['execStr'] = execStr.unicode");
+$values = $jscript->executeString("values", V8Js::FLAG_FORCE_ARRAY);
```

Comment:
The second argument to `executeString` is the script (file) name, not flags (which is the 3rd arg)